ci/repository-request-access.md

```diff
@@ -0,0 +1,121 @@
+---
+title: CoP Repositories Request and Access
+---
+
+The Red Hat Community of Practice makes use of several services to host content, specifically [GitHub](https://github.com/) for source code management and [Quay.io](https://quay.io/) for container images. This guide provide an overview of these tools and how you can make use of them in a self service capacity to request and manage CoP related resources.
+
+## CoP Resource Management Overview
+
+The CoP has established a repository called [org](https://github.com/redhat-cop/org) as the focal point for managing CoP related resources. This repository contains a request management process leveraging GitHub issues along with a set of tools to automate the management of these services.
+
+### Requesting Resources
+
+Request for CoP related services can be made by creating a new [GitHub Issue](https://github.com/redhat-cop/org/issues/new/choose). The following types of resource requests can be made:
+
+* Becoming a CoP Contributor
+* General Support
+* Integration with third party components
+* Permission modification of existing resources
+* Repository management
+* Team management
+
+All resources will be fulfilled by the CoP tools team and tracked in the corresponding issue. See the sections below on how you can leverage some of the automation tools to streamline the request and management process. 
+
```

Comment:
Add (markdown code):

```
> **Important:** even if you have the necessary rights to do so, do not rename/delete repositories and teams without having created an issue for synchronization beforehand. Failing to do so will break our automation.
```
There might be other forbidden though possible actions.

Reply:
Added a warning